google/api/http.proto

```diff
@@ -45,17 +45,18 @@ message Http {
 // gRPC Transcoding is a feature for mapping between a gRPC method and one or
 // more HTTP REST endpoints. It allows developers to build a single API service
 // that supports both gRPC APIs and REST APIs. Many systems, including [Google
-// APIs](https://github.com/googleapis/googleapis),
-// [Cloud Endpoints](https://cloud.google.com/endpoints), [gRPC
```

Comment:
Nit:  this diff would be much easier to evaluate without spurious formatting changes.